cmd/edit/cluster/cmd.go

```diff
@@ -695,6 +705,77 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	var billingAccount string
+	if cmd.Flags().Changed("billing-account") {
+		billingAccount = args.billingAccount
+
+		if billingAccount != "" && !aws.IsHostedCP(cluster) {
+			r.Reporter.Errorf("Billing accounts are only supported for Hosted Control Plane clusters")
+			os.Exit(1)
+		}
+		if billingAccount != "" && !ocm.IsValidAWSAccount(billingAccount) {
+			r.Reporter.Errorf("Billing account number is not valid. Rerun the command with a valid billing account number")
```

Comment:
NP: Might be good to include the billing account number back to the user such as `Billing account number '%s' is not valid...`

Reply:
* If OCM does not bubble this up anyway

Reply:
Also make `Provided billing account number...` would be better, but again not needed

Reply:
Thanks, @hunterkepley I have updated the PR with your suggestions.